src/editor/editor-mathfield.js

```diff
@@ -19,12 +19,11 @@ define([
     'mathlive/core/grapheme-splitter',
     'mathlive/addons/outputLatex', 
     'mathlive/addons/outputMathML', 
-    'mathlive/addons/outputAST', 
     'mathlive/addons/outputSpokenText'], 
     function(Definitions, MathAtom, Lexer, ParserModule, Span, 
     EditableMathlist, MathPath, Keyboard, Undo, Shortcuts, Commands, GraphemeSplitter,
 // eslint-disable-next-line no-unused-vars
-    OutputLatex, OutputMathML, OutputAST, OutputSpokenText) {
```

Comment:
AST support is a work in progress, but I don't see a reason to remove it from here.